modules/storage-persistent-storage-pv.adoc

```diff
@@ -217,7 +217,11 @@ cluster.
 
 |===
 
-The CLI shows the name of the PVC bound to the PV.
+You can view the name of the PVC bound to the PV by running the following command:
+
+----
+$ oc get pvc <pv-claim>
```

Comment:
According to above explanation "You can view the name of the PVC bound to the PV", we want to get PVC name from the bound PV, so the command should be "oc get pv <pv-name>". E.g,
```
$ oc get pv pvc-028e3ee2-e497-4516-98a6-c72a89e88c4b
NAME                                       CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM                                       STORAGECLASS               REASON   AGE
pvc-028e3ee2-e497-4516-98a6-c72a89e88c4b   5Gi        RWO            Delete           Bound    openshift-metering/hive-metastore-db-data   iscsi-targetd-vg-targetd            129m
```
We can get PVC name is hive-metastore-db-data in project openshift-metering.

Reply:
Updated command. @liangxia please verify if LGTM.